plugins/Xpressive/Xpressive.cpp

```diff
@@ -553,7 +553,7 @@ void XpressiveView::expressionChanged() {
 		ExprFront expr(text.constData(),sample_rate);
 		float t=0;
 		const float f=10,key=5,v=0.5;
-		unsigned int i;
+        unsigned int frame_counter = 0;
```

Comment:
Just a note, unsigned int and size_t are similar types in this context and i would recommend you substitute.

Reply:
I said to make the change here too.

Reply:
Please note that `unsigned int` and `size_t` are not trivially exchangeable. See [here](https://stackoverflow.com/a/131833): 

>The `size_t` type [...] is guaranteed to be big enough to contain the size of the biggest object your system can handle [...].
>
>The `size_t` type may be bigger than, equal to, or smaller than an `unsigned int`, and your compiler might make assumptions about it for optimization."